packages/cli/src/build/rollup.ts

```diff
@@ -193,6 +198,15 @@ export let getRollupConfig = (
           },
           preventAssignment: true,
         }),
+      type === "worker" &&
```

Comment:
i feel like `resolve` plugin should also use `browser: true` option when the `type` is worker or browser, WDYT?

Reply:
I can make this change. To be clear, that config is currently only turned on when "type" = "umd". You think I should turn it on for bother "browser" and "worker" as well?

Reply:
yes, i think so - but we should also make sure to use `exportsConditions` appropriately, like here:
https://github.com/cloudflare/durable-objects-rollup-esm/pull/14/files#diff-6814bf77564b4f1c92f5861e184e28fe217c080a047fefa8b73a728f755ec45cR20